### PR TITLE
Reuse bits of old DevTools for NG

### PR DESCRIPTION
**Pull request by piotrswigon**
Creating a parallel webapp for DevTools NG reusing pieces of the original DevTool webapp (http://localhost:8786/devtools/ng.html):

* Message pass-through allowing webSocket, webRTC and Chrome Extension message passing.
* Modular UI with GoldenLayout based panels (for viewing multiple tools at once and customizing the layout).
* Borrowed code from PEC Log to create DevTools NG Raw Log. Now it only renders what can be seen on the screen, which will give better performance longer term. Also reusing some other goodies: search and Download of the log into a file.

Also fixed some constants to follow camelCase.

### CODE REVIEW

File: devtools/src/ng/main.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
```
Comment (SHeimlich) [inline devtools/src/ng/main.js:3]: 2020

File: devtools/src/ng/overview.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
```
Comment (SHeimlich) [inline devtools/src/ng/overview.js:3]: 2020

**Review by SHeimlich — APPROVED** [review]

File: devtools/src/ng/main.js (hunk 1 above)
Comment (piotrswigon) [reply devtools/src/ng/main.js]: Done!

File: devtools/src/ng/overview.js (hunk 2 above)
Comment (piotrswigon) [reply devtools/src/ng/overview.js]: Done!

**Pull request closed without merging**